mysqld-exporter/main.libsonnet

```diff
@@ -9,11 +11,15 @@ local container = k.core.v1.container;
     user,
     host,
     port=3306,
-    image='prom/mysqld-exporter:v0.13.0',
+    image='prom/mysqld-exporter:v0.16.0',
     tlsMode='preferred',
+    configMapName='%s-cfg' % [name],
   ):: {
     local this = self,
-
+    local versionParts = std.split(std.split(image, ':')[1], '.'),
+    local majorVersion = std.parseInt(std.strReplace(versionParts[0], 'v', '')),
```

Comment:
> Added support for **bitnami images** where the tag starts with v.

For my own learning, is this a convention?

Reply:
Sorry, I meant the other way around, bitnami images do not have v while the vanilla mysqld-exporter image has tags starting with v.